core/net/ipv6/uip-nd6.c

```diff
@@ -627,7 +627,8 @@ rs_input(void)
         /* If LL address changed, set neighbor state to stale */
         if(memcmp(&nd6_opt_llao[UIP_ND6_OPT_DATA_OFFSET],
             uip_ds6_nbr_get_ll(nbr), UIP_LLADDR_LEN) != 0) {
-          uip_ds6_nbr_t nbr_data = *nbr;
+          uip_ds6_nbr_t nbr_data;
+          nbr_data = *nbr;
```

Comment:
It's a cc65 limitation.  The compiler code that initiates local variables does the same thing for arrays, structs, and unions.  We cannot assign the contents of array variables to arrays -- directly [we must use memcpy() explicitly].  Therefore, cc65 won't let us assign a struct variable to a struct that is being defined -- we must define, then assign, separately.